### ISSUE TITLE
Publishing issues

### ISSUE THREAD

**Issue by iglocska** [MISP/MISP#979]
It seems that the publishing log entry does not correctly capture the event ID, making the publishing entry not show up in the event history.

Also, exporting, updating a published event and editing the original keeps it published instead of republishing it. An update to an event that results in a published event should always behave as if a publish was triggered.

**Comment by cvandeplas** [comment]
Is this still actual? 
If yes, could you rephrase the second paragraph? In what exact cases (and actions) is the published flag not unset?

**Comment by adulau** [comment]
Tested in recent version and it works. I'll close the issue, feel free to reopen if this is still there.
